### DISCUSSION TITLE
Stand der Dinge

### DISCUSSION THREAD

**Discussion by plaxtika** [SukramJ/aiohomematic#702] [Q&A]
Hallo, ich muss sagen diese Umsetzung ist echt der Hammer :)

Ist schon absehbar wenn es die alte Integration ersetzen wird ? Oder ist das noch nicht im Blick ?

**Comment by danielperna84** [comment]
Am besten mal diese Diskussion lesen: https://github.com/danielperna84/hahomematic/discussions/381
